microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -106,60 +108,61 @@ The following are some of the flags we've seen used.
+| `msWebView2TextureStream` | Allows streaming captured or composed video frames to the WebView2 control, where JavaScript can render or otherwise interact with the frames via W3C standard DOM APIs including the Video element, and MediaStream. |
+| `msWebView2EnableDraggableRegions` | Enables webpages within WebView2 make use of the `app-region: drag\|nodrag` CSS style, which causes elements with that style to behave like a titlebar.  Without this flag, the `app-region: drag\|nodrag` CSS style has no effect. |
+| `msWebView2NativeEventDispatch` | Uses a native mojo connection to dispatch internal events, such as web messages to a renderer process. |
+| `msWebView2SimulateMemoryPressureWhenInactive` | Simulates memory pressure for an inactive WebView. |
+| `msWebView2TreatAppSuspendAsDeviceSuspend` | When all WebViews are suspended, makes WebView2 treat the app as if the device is suspended, and pauses all delayed tasks and timers. |
 | `no-proxy-server` | Overrides any other proxy server flags that are passed. |
 | `net-log-capture-mode` | Sets the granularity of events to capture in the network log.  Valid values: `Default`, `IncludeSensitive`, `Everything`. |
-| `no-network-profile-warning` | If enabled, the browser warns if the UDF is on a network share.  This flag is only supported on Windows; this flag is ignored on other platforms. |
+| `no-network-profile-warning` | Causes the browser to warn if the UDF is on a network share.  This flag is only supported on Windows; this flag is ignored on other platforms. |
 | `no-sandbox` | Disables the sandbox for all process types that are normally sandboxed.  Meant to be used as a browser-level flag for testing purposes only. |
 | `no-first-run` | Skips First-Run tasks, regardless of whether it's actually the First Run, and skips displaying the What's New page.  This flag is overridden by `kForceFirstRun` (for the First-Run Experience (FRE)) and by `kForceWhatsNew` (for displaying What's New).  This flag doesn't drop the First Run sentinel, and thus doesn't prevent the First-Run experience from occurring the next time the Edge WebView browser is launched without this flag.  This flag doesn't update the last What's New milestone, so doesn't prevent What's New from being displayed the next time the Edge WebView browser is launched without this flag. |
-| `PartitionedCookies` | When enabled, sites can opt-in to having their cookies partitioned by the top-level site by using the `Partitioned` attribute.  Partitioned cookies are only sent when the browser is on the same top-level site that it was on when the cookie was set. |
+| `PartitionedCookies` | When enabled,<!-- todo: avoid pattern like "If this flag is enabled," --> sites can opt-in to having their cookies partitioned by the top-level site by using the `Partitioned` attribute.  Partitioned cookies are only sent when the browser is on the same top-level site that it was on when the cookie was set. |
```

Comment:
tried to omit "When enabled," but it was unclear how to give a technically correct statement w/o saying that.
@captainbrosset 
There might be one or rows that have to say "If this flag is enabled,".  This PR moves away from every other row saying that, which produces a conflict across rows.

Reply:
trying argument from simplicity; assume simplicity is an overall net gain in clarity & accuracy.
Result:
`PartitionedCookies` | Enables sites to opt-in to having their cookies partitioned by the top-level site by using the `Partitioned` attribute.  Partitioned cookies are only sent when the browser is on the same top-level site that it was on when the cookie was set.
@captainbrosset